sopel/module.py

```diff
@@ -248,8 +248,10 @@ def add_attribute(function):
     return add_attribute
 
 
-def rate(value):
-    """Decorate a function to limit how often a single user may trigger it.
+def rate(user_rate, channel_rate=0, global_rate=0):
```

Comment:
No particular reason, other than that I don't know what a good default would be. If zero, that's the same as having no @rate decorator at all.